packages/core/packages/PackageManager.ts

```diff
@@ -153,11 +160,17 @@ export class PackageManager {
 	}
 
 	public static async queuePackage(packageName: string, verbose = false) {
-		const command = this.getInstallCommand(this.getManager(), packageName);
-		const packName = packageName.split(/@[^\/]+$/)[0];
-		if (this.getPackageJSON().dependencies[packName] || this.installQueue.find(x => x["packageName"] === packName)) {
+		const command = this.getInstallCommand(this.getManager(), packageName).replace("--save", "--no-save");
```

Comment:
I would go with pass optional parameters to `getInstallCommand` something like:
```
private static getInstallCommand(managerCommand: string, packageName: string, quiet=true, save=true): string
```
or as option:
```
private static getInstallCommand(managerCommand: string, packageName: string, options: InstallCommandOptions): string

interface InstallCommandOptions {
	quiet: boolean;
	save: boolean;
}
```